client/my-sites/customer-home/main.jsx

```diff
@@ -0,0 +1,258 @@
+				/>
+			);
+		}
+
+		const { siteId, hasChecklistData, shouldRenderChecklist } = this.props;
+
+		return (
+			<Main className="customer-home__main is-wide-layout">
+				<PageViewTracker path={ `/home/:site` } title={ translate( 'Customer Home' ) } />
+				<DocumentHead title={ translate( 'Customer Home' ) } />
+				<SidebarNavigation />
+				{ siteId && ! hasChecklistData && <QuerySiteChecklist siteId={ siteId } /> }
+				{ shouldRenderChecklist ? this.renderChecklist() : this.renderCustomerHome() }
+			</Main>
+		);
+	}
+
+	renderChecklist = () => {
+		const { siteId, checklistMode, isJetpackNotAtomic } = this.props;
+
+		return isJetpackNotAtomic ? (
+			<Fragment>
+				<QueryJetpackPlugins siteIds={ [ siteId ] } />
+				<JetpackChecklist />
+			</Fragment>
+		) : (
+			<ChecklistWpcom displayMode={ checklistMode } />
+		);
+	};
+
+	renderCustomerHome = () => {
+		const { translate, customizeUrl, site, siteSlug } = this.props;
+		return (
+			<div className="customer-home__layout">
+				<div className="customer-home__layout-col">
+					<Card>
+						<CardHeading>{ translate( 'My Site' ) }</CardHeading>
+						<h6 className="customer-home__card-subheader">
+							{ translate( 'Review and update my site' ) }
+						</h6>
+						<div className="customer-home__card-button-pair">
+							<Button href={ site.URL } primary>
+								{ translate( 'View Site' ) }
+							</Button>
+							<Button href={ customizeUrl }>{ translate( 'Edit Homepage' ) }</Button>
+						</div>
+					</Card>
+					<Card className="customer-home__card-boxes">
+						<div className="customer-home__boxes">
+							<ActionBox
+								onClick={ () => page( `/page/${ siteSlug }` ) }
+								label={ translate( 'Add a page' ) }
+								iconSrc="/calypso/images/customer-home/page.svg"
+							/>
+							<ActionBox
+								onClick={ () => page( `/post/${ siteSlug }` ) }
+								label={ translate( 'Write blog post' ) }
+								iconSrc="/calypso/images/customer-home/post.svg"
+							/>
+							<ActionBox
+								href={ customizeUrl }
+								label={ translate( 'Customize theme' ) }
+								iconSrc="/calypso/images/customer-home/customize.svg"
+							/>
+							<ActionBox
+								onClick={ () => page( `/themes/${ siteSlug }` ) }
+								label={ translate( 'Change theme' ) }
+								iconSrc="/calypso/images/customer-home/theme.svg"
+							/>
+							<ActionBox
+								href={ customizeUrl }
+								label={ translate( 'Edit menus' ) }
+								iconSrc="/calypso/images/customer-home/menus.svg"
+							/>
+							<ActionBox
+								href="https://support.wordpress.com/images/"
+								label={ translate( 'Change images' ) }
+								iconSrc="/calypso/images/customer-home/images.svg"
+							/>
+							<ActionBox
+								href="https://logojoy.grsm.io/looka"
+								label={ translate( 'Design a logo' ) }
+								iconSrc="/calypso/images/customer-home/logo.svg"
+							/>
+							<ActionBox
+								href={ `https://wordpress.com/email/${ siteSlug }` }
```

Comment:
Just to keep the flows again. :) 

```suggestion
								href={ `/email/${ siteSlug }` }
```

Reply:
Thank you. It has to be `onClick={ () => page( `/email/${ siteSlug }` ) }` though since we'll add something else later, but thank you very much for noticing this.